<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>singleTimer</class>
 <widget class="QDialog" name="singleTimer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSpinBox" name="spinBox_h"/>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_m"/>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_s"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>countUp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>countDown</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>singleTimer</receiver>
   <slot>countupStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>singleTimer</receiver>
   <slot>countdownStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>countupStart()</slot>
  <slot>countdownStart()</slot>
 </slots>
</ui>
